pkg/imrpc/instance.proto

```diff
@@ -0,0 +1,92 @@
+syntax="proto3";
+
+package imrpc;
+
+import "google/protobuf/empty.proto";
+import "imcommon.proto";
+
+service InstanceService {
+    rpc InstanceCreate(InstanceCreateRequest) returns (InstanceResponse) {}
+	rpc InstanceDelete(InstanceDeleteRequest) returns (InstanceResponse) {}
+	rpc InstanceGet(InstanceGetRequest) returns (InstanceResponse) {}
+	rpc InstanceList(google.protobuf.Empty) returns (InstanceListResponse) {}
```

Comment:
nit: ideally we should use the common naming like `ListInstance`; however, I know we juts follow the existing naming used in the current code base, but I would suggest using the common way instead.